### DISCUSSION TITLE
Vector VN1640 & Windows

### DISCUSSION THREAD

**Discussion by Hye993** [collin80/SavvyCAN#799] [General]
Open question to all, but I'm using Savvycan for Windows and want to connect the Vector VN1640 CAN interface.
I've selected QT Serial Bus Devices, select Vectorcan, and then choose the port [bit of guess here, because the port doesn't line up with VectorHardware Screen]. From here I've been running the UDS scanner in conjunction with the Sniffing, to establish if there is any movement on the Bus.

Apologies if this all sounds repetitive and even counter intuitive, I've grown up using Bosch MM6x and Vector Tools, which are clearly engraved into my memory.

The end goal is to read the Bus to get an understanding of what signals are being broadcast publicly, and generate a DBC file.

Any help is appreciated.

**Comment by phophollety** [comment]
Hello, were you able to use the Savvy with the VectorVN? I'm facing the same issue, I wanna use the software but does not want to waste my VN interface.
